WORKSPACE

```diff
@@ -130,15 +130,20 @@ java_grpc_compile()
 load("@graknlabs_build_tools//bazel:dependencies.bzl", "bazel_rules_docker")
 bazel_rules_docker()
 
+##################################
+# Load Distribution dependencies #
+##################################
 
-# TODO: rename the macro we load here to deploy_github_dependencies
-load("@graknlabs_bazel_distribution//github:dependencies.bzl", "github_dependencies_for_deployment")
-github_dependencies_for_deployment()
+load("@graknlabs_bazel_distribution//github:dependencies.bzl", "tcnksm_ghr")
```

Comment:
any chance we can use a slightly more descriptive name for whatever this is??

Reply:
@flyingsilverfin this follows an established convention of naming macros which load Bazel repos from GitHub repositories: `github_org_repo_name`. By that logic, [tcnksm_ghr](https://github.com/tcnksm/ghr) is named like it is 🙂